include/rdma/fabric.h

```diff
@@ -409,8 +409,6 @@ struct fi_alias {
 
 /* control commands */
 enum {
-	FI_GETFIDFLAG,		/* uint64_t flags */
-	FI_SETFIDFLAG,		/* uint64_t flags */
```

Comment:
Removing these from the enum affects the values of the other commands.  These either need to be left or FI_GETOPSFLAG set to 2.